<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableItemProject</class>
 <widget class="QWidget" name="TableItemProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#tableWidgetProject {
	background-color: rgb(255, 255, 255);
	border: 1px solid rgba(0, 0, 0, 80);
	border-radius: 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="tableWidgetProject" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="labelProject">
        <property name="text">
         <string>Project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelIssues">
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 170, 170);</string>
        </property>
        <property name="text">
         <string>Issues</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
